tapir/coop/models.py

```diff
@@ -386,7 +385,7 @@ class ShareOwnership(DurationModelMixin, models.Model):
         max_digits=10,
         decimal_places=2,
     )
-
+    
```

Comment:
```suggestion

```
Unnecessary spaces here.

Reply:
Why? Between all defs is an empty line. And actually this class was made by Théo.

Reply:
It's not about empty line, but unnecessary spaces, i.e. empty line should be empty, here it has spaces :)